src/test/java/com/example/project/test/AETest.java

```diff
@@ -0,0 +1,62 @@
+package com.example.project.test;
+
+import com.example.project.page_object.*;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterSuite;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Test;
+
+import java.util.concurrent.TimeUnit;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class AETest {
+
+    private WebDriver driver;
+
+    @BeforeSuite
```

Comment:
In this specific case I would say that @BeforeClass would be more appropriate here. @BeforeSuite would work for the place which would initialise the driver before all the test classes. But in case it is done on the per class basis I would suggest to stick to the before class.